tests/regression/test_datawarehouse_regression.py

```diff
@@ -109,3 +116,69 @@ def test_get_data_filter_user(valid_dw):
         dataset_type='aggregate',
         filters={'User': '10332'},
     )
+
+
+def test_get_quarter(valid_dw):
+    data = valid_dw.get_data(
+        duration=('2023-01-01', '2023-12-31'),
+        realm='Jobs',
+        metric='CPU Hours: Total',
+        aggregation_unit='Quarter'
+        )
+    __assert_dfs_equal(
+        "jobs-2023-quarters.csv",
+        data,
+        index_col='Time',
+        columns_name='Metric',
+        dtype={'CPU Hours: Total': 'Float64'}
+        )
+
+
+def test_get_years(valid_dw):
```

Comment:
This function is largely the same as the one above it. I would suggest combining them into one pytest parametrized function (using `pytest.mark.parametrize` — the [pytest documentation](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize) explains this and there are also examples in the integration tests). It looks like the differences between the two functions are the duration, the aggregation unit, and the data file, so those would be the parameters. I think the name `test_get_data` would make sense for the function.